universal-login-contracts/contracts/Identity.sol

```diff
@@ -5,8 +5,10 @@ import "./ENSRegistered.sol";
 
 
 contract Identity is KeyHolder, ENSRegistered {
-    constructor(bytes32 _key, bytes32 _hashLabel, string _name, bytes32 _node, ENS ens, FIFSRegistrar registrar, PublicResolver resolver) payable public
+    constructor(
+        bytes32 _key, bytes32 _hashLabel, string _name, bytes32 _node, ENS ens, FIFSRegistrar registrar, PublicResolver resolver, address relayerAddress) 
+        payable public
         ENSRegistered(_hashLabel, _name, _node, ens, registrar, resolver)
-        KeyHolder(_key) {
+        KeyHolder(_key, relayerAddress) {
```

Comment:
I would propose to send tokens back to msg.sender when calling exectuteSigned instead of setting relayer in the constructor.